--- a/config.def.h	2021-07-03 22:30:56.735891581 +0200
+++ b/config.def.h	2021-07-03 22:30:56.735891581 +0200
@@ -5,8 +5,17 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "JetBrains Mono Medium:pixelsize=15:antialias=true:autohint=false";
+static char *font2[] = {
+    "JoyPixels:pixelsize=15:antialias=true:autohint=true",
+    //"JoyPixels:pixelsize=15:antialias=true:autohint=true",
+    "Font Awesome 5 Free Solid:pixelsize=15:antialias=true:autohint=true",
+    "Font Awesome 5 Brands:pixelsize=15:antialias=true:autohint=true",
+    "Symbols Nerd Font:pixelsize=15:antialias=true:autohint=true",
+    "Font Awesome 5 Free Regular:pixelsize=15:antialias=true:autohint=true",
+    "Noto Sans Mono:pixelsize=15:antialias=true:autohint=true",
+};
+static int borderpx = 15;
 
 /*
  * What program is execed by st depends of these precedence rules:
@@ -53,14 +62,14 @@
  * near minlatency, but it waits longer for slow updates to avoid partial draw.
  * low minlatency will tear/flicker more, as it can "detect" idle too early.
  */
-static double minlatency = 8;
+static double minlatency = 4;
 static double maxlatency = 33;
 
 /*
  * blinking timeout (set to 0 to disable blinking) for the terminal blinking
  * attribute.
  */
-static unsigned int blinktimeout = 800;
+static unsigned int blinktimeout = 0;
 
 /*
  * thickness of underline and bar cursors
@@ -96,24 +105,24 @@
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#121212",
+	"#ff5555",
+	"#50fa7b",
+	"#f1fa8c",
+	"#bd93f9",
+    "#ff79c6",
+	"#8be9fd",
+	"#bfbfbf",
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#4d4d4d",
+	"#ff6e67",
+	"#5af78e",
+	"#f4f99d",
+	"#caa9fa",
+	"#ff92d0",
+	"#9aedfe",
+	"#e6e6e6",
 
 	[255] = 0,
 
@@ -139,7 +148,7 @@
  * 6: Bar ("|")
  * 7: Snowman ("☃")
  */
-static unsigned int cursorshape = 2;
+static unsigned int cursorshape = 6;
 
 /*
  * Default columns and rows numbers
@@ -174,7 +183,14 @@
  */
 static MouseShortcut mshortcuts[] = {
 	/* mask                 button   function        argument       release */
-	{ XK_ANY_MOD,           Button2, selpaste,       {.i = 0},      1 },
+
+    // Removing paste with middle click
+	// { XK_ANY_MOD,           Button2, selpaste,       {.i = 0},      1 },
+    // The mod above breaks compatibility with scrollback patch, so the lines
+    // that modify this file are moved here:
+	{ XK_ANY_MOD,           Button4, kscrollup,      {.i = 1},      0, /* !alt */ -1 },
+	{ XK_ANY_MOD,           Button5, kscrolldown,    {.i = 1},      0, /* !alt */ -1 },
+
 	{ ShiftMask,            Button4, ttysend,        {.s = "\033[5;2~"} },
 	{ XK_ANY_MOD,           Button4, ttysend,        {.s = "\031"} },
 	{ ShiftMask,            Button5, ttysend,        {.s = "\033[6;2~"} },
